spec/fixtures/files/datahashes/core/exhibition1.json

```diff
@@ -26,19 +26,19 @@
     "exhibitionPersonPersonUlan": "We go",
     "exhibitionPersonOrganizationLocal": "breakup",
     "exhibitionPersonOrganizationUlan": "Again",
-    "exhibitionPersonRole": "Educator;Public programs coordinator;",
+    "exhibitionPersonRole": "Educator^^^^Public programs coordinator",
     "planningNote": "Perfect condition",
     "curatorialNote": "Anayways",
     "generalNote": "Need me right here",
     "boilerplateText": "When it's up and down",
     "galleryRotationName": "Toppa;Concious",
     "galleryRotationStartDateGroup": null,
     "galleryRotationEndDateGroup": null,
-    "galleryRotationNote": "Anything",
+    "galleryRotationNote": "Eligible;Anything",
     "exhibitionReferenceLocal": "Wanting",
     "exhibitionReferenceWorldcat": "Patiently",
-    "exhibitionReferenceType": "Press release;News article",
-    "exhibitionReferenceNote": "Cuddle;Clap",
+    "exhibitionReferenceType": "News article;Press release",
+    "exhibitionReferenceNote": "Clap;Cuddle",
```

Comment:
Changed the order of rows due to limitation on the order in which mapper processes authorities in multi-authority fields